fr.masson_informatique.WhosWho.yml

```diff
@@ -0,0 +1,241 @@
+    - "/share/opencv4/haarcascades/haarcascade_fullbody.xml"
+    - "/share/opencv4/haarcascades/haarcascade_lefteye_2splits.xml"
+    - "/share/opencv4/haarcascades/haarcascade_licence_plate_rus_16stages.xml"
+    - "/share/opencv4/haarcascades/haarcascade_lowerbody.xml"
+    - "/share/opencv4/haarcascades/haarcascade_profileface.xml"
+    - "/share/opencv4/haarcascades/haarcascade_righteye_2splits.xml"
+    - "/share/opencv4/haarcascades/haarcascade_russian_plate_number.xml"
+    - "/share/opencv4/haarcascades/haarcascade_smile.xml"
+    - "/share/opencv4/haarcascades/haarcascade_upperbody.xml"
+    - "/share/opencv4/lbpcascades/lbpcascade_frontalcatface.xml"
+    - "/share/opencv4/lbpcascades/lbpcascade_frontalface_improved.xml"
+    - "/share/opencv4/lbpcascades/lbpcascade_frontalface.xml"
+    - "/share/opencv4/lbpcascades/lbpcascade_profileface.xml"
+    - "/share/opencv4/lbpcascades/lbpcascade_silverware.xml"
+    - "/share/opencv4/valgrind.supp"
+    - "/share/opencv4/valgrind_3rdparty.supp"
+
+  - name: python3-Pillow
+    buildsystem: simple
+    build-commands:
+    - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+      --prefix=${FLATPAK_DEST} "Pillow" --no-build-isolation
+    sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/43/6e/59853546226ee6200f9ba6e574d11604b60ad0754d2cbd1c8f3246b70418/Pillow-9.1.1.tar.gz
+      sha256: 7502539939b53d7565f3d11d87c78e7ec900d3c72945d4ee0e2f250d598309a0
+
+  - name: python3-Willow
+    buildsystem: simple
+    build-commands:
+    - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+      --prefix=${FLATPAK_DEST} "Willow" --no-build-isolation
+    sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/b5/bb/e654a03c8224b06f9e12381cd0c67028f460e625df55711f7ded30afcaf1/Willow-1.4.1-py2.py3-none-any.whl
+      sha256: fc4042696d090e75aef922fa1ed26d483c764f005b36cf523cf7c34e69d5dd7a
+
+  - name: imagemagick
+    buildsystem: autotools
+    config-opts:
+    - "--enable-shared=no"
+    - "--enable-static=yes"
+    - "--disable-docs"
+    - "--without-magick-plus-plus"
+    - "--without-perl"
+    - "--without-bzlib"
+    - "--without-dps"
+    - "--without-fpx"
+    - "--without-jbig"
+    - "--without-jp2"
+    - "--without-lcms"
+    - "--without-lzma"
+    - "--without-png"
+    - "--without-tiff"
+    - "--without-wmf"
+    - "--without-xml"
+    - "--without-zlib"
+    - "--without-x"
+    sources:
+    - type: archive
+      url: https://download.imagemagick.org/ImageMagick/download/releases/ImageMagick-7.1.0-35.tar.xz
+      sha256: 26063653de155bab820894e5216a38577420c4dfd59e784daca6601e37f294e0
+    cleanup:
+    - "/bin/MagickCore-config"
+    - "/bin/MagickWand-config"
+
+cleanup:
+  - "/bin/distro"
+  - "/include/*"
+  - "/lib/debug/*"
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=xdg-run/gvfsd
```

Comment:
```suggestion
```

Are you sure these are needed for anything?

Reply:
I am absolutely unsure :-)
This software uses file chooser portal so that user can choose directories to read and files to write. As these can be located on a GVFS mount, I thought these permissions were necessary.